doc/versioned-translated.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/sh
+
+current_branch=$(git rev-parse --abbrev-ref HEAD)
+
+cd $(git rev-parse --show-toplevel)
+
+##############
+# BUILD DOCS #
+##############
+  
+# first, cleanup any old builds' static assets
+# make -C doc clean
+rm -rf doc/build/html
+rm -rf doc/build/gettext
```

Comment:
```suggestion
rm -rf doc/build
```

Any reason not to just delete the build directory? Also whats the purpose of the commented line?

Reply:
Right, I made the change